cmd/generate/cmd_util.go

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/nginxinc/nginx-go-crossplane/internal/generator"
+)
+
+type filterFlag struct {
+	filter map[string]struct{}
+}
+
+func (reader *filterFlag) Set(value string) error {
+	if reader.filter == nil {
+		reader.filter = make(map[string]struct{})
+	}
+	reader.filter[value] = struct{}{}
+	return nil
+}
+
+func (reader *filterFlag) String() string {
+	return fmt.Sprintf("%v", *reader)
+}
```

Comment:
Nit: no longer a "Reader"

```suggestion
func (f *filterFlag) Set(value string) error {
	if f.filter == nil {
		f.filter = make(map[string]struct{})
	}
	f.filter[value] = struct{}{}
	return nil
}

func (f *filterFlag) String() string {
	return fmt.Sprintf("%v", *f)
}
```

Reply:
Sure! Applied. Should've checked all the usages after rename...